airbyte-cdk/bulk/toolkits/extract-jdbc/src/test/resources/h2source/expected-messages-stream-warm-start.json

```diff
@@ -57,7 +57,7 @@
         }
       },
       "sourceStats": {
-        "recordCount": 2.0
+        "recordCount": 1.0
```

Comment:
this was wrong but we never noticed because we don't look at the expected state message too closely